rest-server/src/middlewares/parameter.js

```diff
@@ -35,6 +35,7 @@ const validate = (schema) => {
           message: errorMessage,
         });
       } else {
+        req.original_body = req.body;
         req.body = value;
         next();
```

Comment:
What changes will made to new body compare to the original body after the validation?

Reply:
The following fields will be added if not explicitly specified: `portList`, `username`, `dataDir`, `outputDir`, `codeDir`, `gpuType`, `virtualCluster`, and `retryCount`. I don't think we should tell users that we have auto-completed and altered their original request.